int64 prediction_time
int64 order
float64 x_min
float64 x_max
float64 y_min
float64 y_max
float64 cell_size
---
STeFMapMsg stefmap
